src/libs/tools/include/plugins.hpp

```diff
@@ -47,7 +38,7 @@ struct Place
 class Plugins
 {
 protected:
-	std::vector<Plugin *> plugins;
+	std::vector<Slot *> plugins;
```

Comment:
@kodebach obviously this is a vector of dynamically allocated objects. Why does this class not have a destructor? I get tons of leaks for the `Plugins` class.

Reply:
I have no idea... @markus2330 Do you know?

Reply:
I've added one, it fixes the leaks.

Reply:
I reverted the commit that added the destructor. I don't think it made things better (basically every test using the KDB had a memory issue in the last build). I'll see if I can find the issue. But since this is C++, there is probably a solution without writing the destructor ourselves.

Reply:
Well this added back the memleaks for me :(. Better refactor it to not use a linked list then.

Reply:
I'm looking into it. I think part it worked, before because the `Plugin` class has a destructor and that was called automatically by `std::vector`. By inserting a `elektraMalloc`'d list inbetween we broke that